ecs-agent/acs/session/session.go

```diff
@@ -268,7 +268,7 @@ func (s *session) startSessionOnce(ctx context.Context) error {
 		})
 		return err
 	}
-	s.metricsFactory.New(metrics.ACSSessionCallDurationName).WithGauge(time.Since(acsConnectionStartTime).Nanoseconds()).Done(nil)
+	s.metricsFactory.New(metrics.ACSSessionCallDurationName).WithGauge(time.Since(acsConnectionStartTime).Milliseconds()).Done(nil)
 	defer disconnectTimer.Stop()
```

Comment:
nit (non-blocking): `metrics.ACSSessionCallDurationName` currently evaluates to `ACSSession.ACSConnectDuration`. Consider renaming such that it evaluates to something like `ACSSession.ACSConnectDurationInMilliseconds` so that it is clear to readers what the duration units are. 

(this same comment applies to `metrics.DiscoverPollEndpointDurationName`)